src/Orleans.Core/Streams/QueueAdapters/IBatchContainerBatch.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace Orleans.Streams
+{
+    /// <summary>
+    /// A batch of queue messages (see IBatchContainer for description of batch contents)
+    /// </summary>
+    public interface IBatchContainerBatch : IBatchContainer
```

Comment:
Who uses this interface?

Reply:
BatchContainerBatch implements the interface.

Reply:
Yes, but what calls the interface.  If nothing calls it, then it is unnecessary as an abstraction, correct?

Reply:
It was previously used for logic in code that was going to send batches to the application layer. But since I'm reverting that change, I'll remove this interface.